### ISSUE TITLE
Allow maximizing the 1:1 (video) call "widget"

### ISSUE THREAD

**Issue by HarHarLinks** [vector-im/element-web#22318]
#### What would you like to do?
Element has gained the ability to not only pin widgets but also "maximize" them, i.e. replace the timeline and display the timeline in the right panel location.
I would like to do this with 1:1 native video calls.

#### Why would you like to do it?
sometimes you want to share some notes

#### How would you like to achieve it?
just reuse the maximized widgets

**Comment by justjanne** [comment]
Video rooms offer a UI related to what you’re describing already, though they’re obviously meant for group calls over longer timeframes.

**Comment by HarHarLinks** [comment]
Exactly, it would be great to be able to use this same UI for "non-persistent" normal calls.
Actually, the way that the call widget is currently "pinned" but larger than normal pinned widgets makes me think that the design intention is to use maximized widgets here (but those didn't exist when calls were introduced).
